### ISSUE TITLE
[DNNL] Implement operator softmax with cross entropy

### ISSUE THREAD

**Issue by jczaja** [PaddlePaddle/Paddle#23073]
DNNL 1.2 released operator softmax with log loss . Perhaps it can be used to implement softax with cross entropy

**Comment by jczaja** [comment]
low priority for now
